_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,140 @@
+    - Beginner tutorials: these sessions are for current and future OGGM users to learn how to set-up and customize a model run.
+    - Advanced tutorials: on topics such as <i>How to contribute to OGGM</i>, <i>use OGGM with your own data</i>, or <i>use parts of OGGM with another model</i>.
+    - Q&A session: on model problems and possible solutions.
+4.  <b>Workshop excursion</b>: Ice sheets covered all of Scotland in the past, leaving an amazing landscape behind. We plan to have a one-day excursion to one of Scotland's iconic ice carved landscapes.
+
+We like to keep the schedule flexible in order to meet the needs, interests and suggestions 
+of the participants. Tentatively, we plan to have presentations mostly in the mornings and 
+group discussions and hand-on sessions in the afternoons. 
+
+### Who can participate ?
+Anyone working on glaciological modeling using OGGM,
+but also any other global glacier modeler keen on learning more about OGGM! In particular, 
+we would like to encourage model users (current and future) as well as model developers to join 
+us. The number of participants is limited to approximately 30 due to the housing limitations 
+at the University accomodation. Therefore users and developers of OGGM will be given 
+priority in case interest exceeds capacity.
+
+### Location
+The workshop will be held at the [Edinburgh Climate Change Institute (ECCI)](https://edinburghcentre.org/). ECCI is part of the School of GeoSciences, University of Edinburgh. Located in the [heart of the city,](https://goo.gl/maps/Edb3ixpE4D745icW8) is a leading centre for research, teaching, policy and practice for climate action. In general, summers in Edinburgh are reasonably warm and pleasant and the city has a lot to offer including one of the most famouse summer festivals in the world [The Fringe](https://www.edfringe.com/experience/what-is-the-festival-fringe). 
+
+### Transportation
+Edinburgh is the capital city of Scotland (in the UK) therefore, it is very accesible by flight or train. Edinburgh Airport is the closest international airport and Edinburgh Waverley is the name of the central train station. From Amsterdam (NL) there are a regular ferries to Newcastle (England), which is only approx. 2hrs by train from Edinburgh. There are also daily trains from London. 
+
+From the Airport there are several bus routes and tram to the city centre ([Princes street](https://goo.gl/maps/2to6fG8Mgj7DtYQk6)), more information on public transport to and from the Airport can be found [here](https://www.edinburghairport.com/transport-links). Visit the [Lothian buses website](https://www.lothianbuses.com/) for transport information within Edinburgh. 
+
+From the Edinburgh Waverley train station to ECCI there is approx. a 8 min walk distance. And the workshop accommodation is reachable by bus.
+
+
+### Arrival / departure dates
+The workshop starts on the 28th September in the morning and ends on the 1st of September in the afternoon. ??
```

Comment:
Yes

Reply:
We recommend to arrive on Sunday (beginners) or Monday (all participants) and depart on Friday evening or Saturday morning.